ci/ray_ci/BUILD.bazel

```diff
@@ -34,6 +34,12 @@ py_binary(
     deps = [":ray_ci_lib"],
 )
 
+py_binary(
+    name = "move_test_state",
+    srcs = ["move_test_state.py"],
+    deps = [":ray_ci_lib"],
```

Comment:
just move this `py_binary` into the subdir, and declare rayci lib as deps?

don't include the script in the lib's source file list.